content/en.json

```diff
@@ -143,16 +136,16 @@
       ]
     },
     "technology": {
-      "title": "Used Technology",
+      "title": "Technology",
```

Comment:
Ooh slipstreaming a change to one of the pillar names.... good change though, I'd say! 👍🏻